Sources/MySQL/SQL/MySQLAlterTable.swift

```diff
@@ -67,7 +74,10 @@ public struct MySQLAlterTable: SQLAlterTable {
                 return sql
             }
         } + constraints.map { "ADD " + $0.serialize(&binds) }
-        sql.append(actions.joined(separator: ", "))
+        let deleteActions = deleteColumns.map {
+            return "DROP `" + $0.identifier.string + "`"
```

Comment:
`$0.serialize(&binds)` should be used here instead of manually escaping using back-ticks. An example of this exists in the lines above.